docs/en/stack/ml/nlp/ml-nlp-deploy-models.asciidoc

```diff
@@ -84,6 +89,40 @@ eland_import_hub_model
 
 For more details, refer to https://github.com/elastic/eland#nlp-with-pytorch.
 
+[discrete]
+[[ml-nlp-import-docker]]
+=== Import with Docker
+
+IMPORTANT: To use the Docker container, you need to clone the Eland repository: 
+https://github.com/elastic/eland
+
+If you want to use Eland without installing it, build the Docker container to 
+run the available scripts:
+
+```bash
+$ docker build -t elastic/eland .
+```
+
+You can now use the container interactively:
+
+```bash
+$ docker run -it --rm --network host elastic/eland
+```
+
+Running installed scripts is also possible without an interactive shell:
+
+```bash
+docker run -it --rm elastic/eland \
+    eland_import_hub_model \
+      --url $ELASTICSEARCH_URL \
+      --hub-model-id elastic/distilbert-base-uncased-finetuned-conll03-english \
+      --start
+```
+
+Replace the `$ELASTICSEARCH_URL` with the URL for your {es} cluster. For 
+authentication purposes, include an administrator username and password in the 
```

Comment:
Instead of the second sentence can you say something like "see the Authentication methods section for details"

Reply:
Addressed via [03f4002](https://github.com/elastic/stack-docs/pull/2418/commits/03f4002e8fa49dbf7ff7805a2c7996e252e3ca03).